cxx-qt-gen/src/syntax/attribute.rs

```diff
@@ -135,5 +186,55 @@ mod tests {
         assert_eq!(attribute_tokens_to_list(&module.attrs[4]).unwrap()[0], "A");
         assert_eq!(attribute_tokens_to_list(&module.attrs[4]).unwrap()[1], "B");
         assert_eq!(attribute_tokens_to_list(&module.attrs[4]).unwrap()[2], "C");
+        assert!(attribute_tokens_to_list(&module.attrs[5]).is_ok());
+        assert_eq!(attribute_tokens_to_list(&module.attrs[5]).unwrap().len(), 0);
+    }
+
+    #[test]
+    fn test_attribute_tokens_to_map() {
+        let module: ItemMod = tokens_to_syn(quote! {
+            #[invokable]
+            #[cxx_qt::bridge]
+            #[cxx_qt::signals(MyObject)]
+            #[cxx_qt::bridge(namespace = "my::namespace")]
+            #[cxx_qt::list(A, B, C)]
+            #[cxx_qt::bridge(a = "b", namespace = "my::namespace")]
+            #[cxx_qt::bridge(a = "b", namespace = "my::namespace", namespace = "my::namespace")]
```

Comment:
I know this is just testing the syntax parsing, but in the actual parser I think it should return an error if multiple namespaces are specified.

Reply:
This test is checking that that case errors, `assert!(attribute_tokens_to_map::<Ident, LitStr>(&module.attrs[6]).is_err());`